preprocessing/cameras/create_missing_video_segments_from_raw_videos.py

```diff
@@ -118,14 +77,14 @@ def subprocess_run_with_guardfile(cmd: list[str], guardfile_path: Path):
                         parents=True, exist_ok=True
                     )
                     # Reference from /mnt/staff-bulk/ewi/insy/SPCDataSets/conflab-mm/raw/video/overhead/cam04/videoRot.sbatch
-                    cmd = [ 
+                    cmd = [
```

Comment:
Do you want to do the formatting of the command here as well as I suggested above?